drivers/serial/uart_stm32.c

```diff
@@ -455,7 +456,10 @@ static void uart_stm32_poll_out(const struct device *dev,
 	while (!LL_USART_IsActiveFlag_TXE(UartInstance)) {
 	}
 
-	LL_USART_ClearFlag_TC(UartInstance);
+#ifdef CONFIG_PM
```

Comment:
We should be able to remove these, thanks to https://github.com/zephyrproject-rtos/zephyr/pull/38472